workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/metrics.py

```diff
@@ -90,6 +110,31 @@ def rmse_3day(diags):
     return rms_at_day_3
 
 
+@add_to_metrics("rmse_days_3to7_avg")
+def rmse_days_3to7_avg(diags):
+    rms_global = grab_diag(diags, "rms_global").drop(GRID_VARS, errors="ignore")
+    time_since_start = rms_global.time.values - rms_global.time.isel(time=0).item()
+
+    ds = rms_global.assign_coords(
+        {
+            "days_since_start": (
+                "time",
+                [t.item() / ns_per_day for t in time_since_start],
+            )
+        }
+    )
+
+    if max(ds["days_since_start"] > 8):
+        rmse_days_3to7_avg = (
+            ds.where(ds["days_since_start"] >= 3).where(ds["days_since_start"] <= 8)
+        ).mean(skipna=True)
+    else:  # don't compute metric if run didn't make it to 7 days
+        rmse_days_3to7_avg = xr.Dataset()
```

Comment:
```suggestion
    rms_global_daily = rms_global.resample(time="1D").mean()

    if rms_global_daily.sizes["time"] >= 7:
        rmse_days_3to7_avg = rms_global_daily.isel(time=slice(2, 7)).mean("time")
    else:  # don't compute metric if run didn't make it to 7 days
        rmse_days_3to7_avg = xr.Dataset()
```
following the pattern in the `rmse_day3` function makes this simpler, although I guess if the start time is not at 0 UTC it would not be quite correct. But I think it's best to be consistent with the other function.

Reply:
Would be good to double-check my slice integers here.